android/filament-android/build.gradle

```diff
@@ -52,6 +52,7 @@ android {
                 arguments.add("-DANDROID_PIE=ON")
                 arguments.add("-DANDROID_PLATFORM=android-21")
                 arguments.add("-DANDROID_STL=c++_static")
+                arguments.add("-DANDROID_ABI=armeabi-v7a")
```

Comment:
this limits building to 32 bits. emscriptten expects 32 bits. you'll get static assert warnings if you try to target a 64 bit binary.

note: I think this isn't the right way to do this, as it builds to a misleading folder name.